parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetFileReader.java

```diff
@@ -517,7 +524,74 @@ public PageReadStore readNextRowGroup() throws IOException {
     return columnChunkPageReadStore;
   }
 
+  /**
+   * Reads the requested predicate columns dictionaries from the row group at the current file position.
+   * Not reading all the requested columns, only read predicate columns.
+   * eg. select columnA, columnB, columnC from table where columnC > 10
+   *     only reads columnC's dictionary
+   * @param requestedSchema the predicate columns to read
+   * @throws IOException if an error occurs while reading
+   * @return the DictionaryPageReadStore which can provide DictionaryPageReaders for each column.
+   */
+  public DictionaryPageReadStore readNextRowGroupDictionaries(MessageType requestedSchema) throws IOException {
```

Comment:
@zhenxiao Isn't this name a little misleading?  I believe this is better as `getCurrentRowGroupDictionaries(...)` because it just returns the current row group and doesn't advance.